ibm/data_source_ibm_container_cluster.go

```diff
@@ -105,11 +145,35 @@ func dataSourceIBMContainerClusterRead(d *schema.ResourceData, meta interface{})
 		boundedService["namespace"] = service.Namespace
 		boundedServices = append(boundedServices, boundedService)
 	}
-
+	//Set Vlans
+	vlansR := clusterFields.Vlans
+	vlans := make([]map[string]interface{}, 0, len(vlansR))
```

Comment:
Since we already know the length, using make([]map[string]interface{}, len(vlansR)) and then vlans[i] = ... is better I guess

Reply:
On second though.. how about pulling this thing in a function and calling it flattenVlans in the strcutures.go

Reply:
Done